streams/src/test/java/org/apache/kafka/streams/tools/StreamsResetterTest.java

```diff
@@ -76,6 +78,35 @@ public void testResetToSpecificOffsetWhenBetweenBeginningAndEndOffset() {
         assertEquals(3, records.count());
     }
 
+    @Test
+    public void testResetOffsetToSpecificOffsetWhenAfterEndOffset() {
+        final long[] beginningOffsets = {0L, 5L, 10L, 20L};
+        final long[] endOffsets = {0L, 5L, 10L, 20L};
+        // Test on multiple beginning and end offset combinations
+        for (int beginningOffsetIndex = 0; beginningOffsetIndex < beginningOffsets.length; beginningOffsetIndex++) {
```

Comment:
I actually think that a single combination beginning=5 end=10 should be sufficient.

Reply:
Sure. Will simplify then